test/oni2-keyboard-layout/KeymapTest.re

```diff
@@ -0,0 +1,22 @@
+open Oni2_KeyboardLayout;
+open TestFramework;
+
+describe("Keymap", ({test, _}) => {
+  test("getCurrentKeymap", ({expect, _}) => {
+    let keymap = Keymap.getCurrent();
+
+    expect.bool(Keymap.size(keymap) > 0).toBeTrue();
+  });
+
+  test("entryOfKey", ({expect, _}) => {
+    let keymap = Keymap.getCurrent();
+    let entry = Keymap.entryOfKey(keymap, "KeyS");
+
+    expect.option(entry).toBeSome();
+
+    let entry = Option.get(entry);
+
+    expect.string(Keymap.(entry.unmodified)).toEqual("s");
+    expect.string(Keymap.(entry.withShift)).toEqual("S");
+  });
+});
```

Comment:
@bryphe I got the keymaps working on macOS, which I think is the feature you most wanted. This test will need to be improved since it obviously will only work when the S key is mapped to S, but for now it works as a small test

Reply:
Awesome! Are you planning on looking at the Linux / Windows bindings too? Or should I check those out?

Reply:
I'll look at those as well -- macOS is always the easiest first step for me since it's what my host is, but I will definitely look into Windows and Linux as well!

Reply:
Cool! Looks like CI is failing due to missing symbols on Windows / Linux - I guess we'll need at least stubs for this to get green